index.html

```diff
@@ -2238,6 +2451,24 @@ <h2>
         </p>
       </div>
     </section>
+    <section>
+      <h3>
+        Glossary
```

Comment:
This should really be part of a Model section... or wherever the general idea of gamepad is defined.